### PR TITLE
Copy from node-build with code-coverage

### PR DESCRIPTION
**Pull request by RolfDeVries**
Attempt 2 for codecoverage, this time non-breaking:

Copy of node-build renamed to front-end-build
Upgrade to latest stable debian image: buster
Added browsers and sonar-scanner to docker image
Add npm test:prod

### CODE REVIEW

File: front-end-build.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        echo -e "\n@inforit:registry=https://www.myget.org/F/inforit/npm/\n//www.myget.org/F/inforit/npm/:_authToken=$MYGET_ACCESS_TOKEN" >> ~/.npmrc
+        echo "@inforit scope sucesfully registered"
+    else
+        echo "@inforit scope already registered"
+    fi
+fi
+
+rm -rf "$DIST"
+
+npm install
```
Comment (Mastermindzh) [inline front-end-build.sh:32]: For safety I would add "--if-present" to build, test and zipdist.

**Review by Mastermindzh — APPROVED** [review]
Looks good. Few small things:

.editorconfig
.gitignore
.dockerignore

are not the "standard" inforit ones. 
We only ever described this on Confluence but I will make a code convention repo so it's more clear.

Other than that it looks really solid now :smile:

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
```
[the remark this answers is not in the document]
Comment (RolfDeVries) [reply .editorconfig:27]: Was this repo forked from an inforit base? If so, should be also update that repo?

File: .editorconfig [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = false
+insert_final_newline = false
```
[the remark this answers is not in the document]
Comment (RolfDeVries) [reply .editorconfig]: Was this repo forked from an inforit base? If so, should be also update that repo?

File: front-end-build.sh (hunk 1 above)
Comment (RolfDeVries) [reply front-end-build.sh:32]: done

File: .editorconfig (hunk 3 above)
[the remark this answers is not in the document]
Comment (Mastermindzh) [reply .editorconfig]: Yes. Quite right. I will do that today :)

**Pull request merged**